From 65bb2d954b1db0ee3524e7f8d6d5121807af8ffe Mon Sep 17 00:00:00 2001
From: Trent Mick <trentm@gmail.com>
Date: Wed, 28 Aug 2019 08:43:52 -0700
Subject: [PATCH] joyent/node-triton#266 v7.5.0 Reviewed by: Cody Peter Mello
 <melloc@writev.io> Approved by: Cody Peter Mello <melloc@writev.io>

---
 CHANGES.md   | 12 ++++++++++++
 package.json |  2 +-
 2 files changed, 13 insertions(+), 1 deletion(-)

diff --git a/CHANGES.md b/CHANGES.md
index b822260..7ef050e 100644
--- a/CHANGES.md
+++ b/CHANGES.md
@@ -6,6 +6,18 @@ Known issues:
 
 ## not yet released
 
+(nothing yet)
+
+## 7.5.0
+
+- [joyent/node-triton#264] Update `triton rbac role[s]` to use the new
+  CloudAPI role API (see https://apidocs.joyent.com/cloudapi/#900).
+
+- [joyent/node-triton#265] Un-hide the `triton --act-as ACCOUNT ...`
+  option (and add a matching `TRITON_ACT_AS=ACCCOUNT` envvar) that allows
+  one to masquerade as the given account. This is useful (a) for operator
+  accounts and (b) members of cross-account roles.
+
 ## 7.4.1
 
 - [TRITON-1869] node-triton needs `triton instance migration finalize`.
diff --git a/package.json b/package.json
index ad01b1b..81f74b8 100644
--- a/package.json
+++ b/package.json
@@ -1,7 +1,7 @@
 {
   "name": "triton",
   "description": "Joyent Triton CLI and client (https://www.joyent.com/triton)",
-  "version": "7.4.1",
+  "version": "7.5.0",
   "author": "Joyent (joyent.com)",
   "homepage": "https://github.com/joyent/node-triton",
   "dependencies": {
-- 
2.21.0

